test/unit/state/frameStateUtilTest.js

```diff
@@ -17,92 +17,6 @@ describe('frameStateUtil', function () {
     this.windowState = Immutable.fromJS(Object.assign({}, defaultWindowStore.toJS()))
   })
 
-  describe('cloneFrame', function () {
-    before(function () {
-      this.frameOpts = {
-        key: 1,
-        tabId: 2,
-        parentFrameKey: 3,
-        audioMuted: true,
-        canGoBack: true,
-        canGoForward: true,
-        icon: 'icon',
-        title: 'title',
-        isPrivate: true,
-        partitionNumber: 4,
-        themeColor: 'ffffff',
-        computedThemeColor: 'aaaaaa',
-        history: ['http://brave.com'],
```

Comment:
do we still need this now that we have history methods on the webview/webcontents? Are we still using it anywhere?